src/test/compile-fail/lint-stability.rs

```diff
@@ -13,7 +13,7 @@
 // aux-build:stability_cfg1.rs
 // aux-build:stability_cfg2.rs
 
-#![deny(deprecated)]
+#![allow(deprecated)]
```

Comment:
I had to split it in two. It's that thing where if we kept going we'd have to fix so many warnings from lints that were never taken into account. Here the non-lint stability case was moved before typeck.